src/main/java/com/academia/library/eception/BookNotFoundException.java

```diff
@@ -0,0 +1,8 @@
+package com.academia.library.eception;
+
+public class BookNotFoundException extends RuntimeException{
+
+    public BookNotFoundException(String message){
+        super(message);
```

Comment:
It is totally useless. Use something more relevant to this case

Reply:
Ok, thank